dashboards/persistentvolumesusage.libsonnet

```diff
@@ -0,0 +1,110 @@
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+local promgrafonnet = import '../lib/promgrafonnet/promgrafonnet.libsonnet';
+local numbersinglestat = promgrafonnet.numbersinglestat;
+local gauge = promgrafonnet.gauge;
+
+{
+  grafanaDashboards+:: {
+    'persistentvolumesusage.json':
+      local sizeGraph = graphPanel.new(
+        'Volume size usage',
+        datasource='$datasource',
+        format='percent',
+        max=100,
+        min=0,
+        span=12,
+        legend_show=true,
+        legend_values=true,
+        legend_min=true,
+        legend_max=true,
+        legend_current=true,
+        legend_total=false,
+        legend_avg=true,
+        legend_alignAsTable=false,
+        legend_rightSide=false,
+      ).addTarget(prometheus.target(
+        |||
+          (kubelet_volume_stats_capacity_bytes{%(kubeletSelector)s, persistentvolumeclaim="$volume"} - kubelet_volume_stats_available_bytes{%(kubeletSelector)s, persistentvolumeclaim="$volume"}) / kubelet_volume_stats_capacity_bytes{%(kubeletSelector)s, persistentvolumeclaim="$volume"} * 100
+        ||| % $._config,
+        legendFormat='{{ Usage }}',
+        intervalFactor=1,
+      ));
+
+      local inodesGraph = graphPanel.new(
+        'Volume inodes usage',
+        datasource='$datasource',
+        format='percent',
+        max=100,
+        min=0,
+        span=12,
+        legend_show=true,
+        legend_values=true,
+        legend_min=true,
+        legend_max=true,
+        legend_current=true,
+        legend_total=false,
+        legend_avg=true,
+        legend_alignAsTable=false,
+        legend_rightSide=false,
+      ).addTarget(prometheus.target(
+        |||
+          kubelet_volume_stats_inodes_used{%(kubeletSelector)s, persistentvolumeclaim="$volume"} / kubelet_volume_stats_inodes{%(kubeletSelector)s, persistentvolumeclaim="$volume"} * 100
+        ||| % $._config,
+        legendFormat='{{ Usage }}',
+        intervalFactor=1,
+      ));
+
+      dashboard.new(
+        'Kubernetes Persistent Volumes usage',
```

Comment:
We currently have dashboards called `Node`, `Pods` and so on, or we prefix them with `K8s / ...`. We should be consistent and call this one `Persistent Volumes`.